### ISSUE TITLE
tsr-detect-non-literal-fs-filename gets triggered when __dirname is used

### ISSUE THREAD

**Issue by bcko** [webschik/tslint-config-security#13]
tsr-detect-non-literal-fs-filename error gets triggered when nodejs __dirname is used in `fs`

```javascript
const variableName = fs.readFileSync(__dirname + 'filename.txt`, 'utf-8')
```

Do you want me to make a pull request? 
Thank you.

**Comment by webschik** [comment]
Fixed in `v1.14.0`

**Comment by bcko** [comment]
thank you @webschik !
